src/components/header/_header.scss

```diff
@@ -28,34 +28,41 @@
     @include mq(xs) {
       height: 46px;
     }
+    .header__logo--small > {
```

Comment:
It looks to me like, if you put `display: block` on `header__logo-link` and on the `svg`, you could remove all these padding adjustments